packages/dd-trace/src/tracer.js

```diff
@@ -138,6 +138,14 @@ class DatadogTracer extends Tracer {
 
     return this
   }
+
+  getLocalRootSpan () {
```

Comment:
We should not expose this as it's a foot gun. If there is a use case for adding data to the root span other options should be explored. There are no guarantees when it comes to the root span still being active.

Reply:
That makes sense and I believe this was your concern on similar requests - https://github.com/DataDog/dd-trace-js/issues/725#issuecomment-738213349. 

It sounds like this isn't functionality we should be exposing due to memory concerns. I can go ahead and close this PR. Thanks @rochdev  🙇

Reply:
@rochdev Nothing specific or urgent. I have seen a decent amount of request for this kind of functionality on tickets and noticed it on the backlog. I agree that my current approach could lead to memory concerns as the root span may not be active.

Reply:
I've seen it a lot too, but the actual requirement is adding data to the _trace_, not necessarily accessing the local "root" span, especially since it might have been flushed already when you need it. It's probably worth it to start an effort to support trace-level metadata instead which would support a lot more use cases.

Reply:
@rochdev has there been any movement on supporting trace-level (or even "resource" level) tags? Is there an issue or ticket I can be following?